python/src/opendp/_convert.py

```diff
@@ -391,6 +397,14 @@ def _tuple_to_slice(val: Tuple[Any, ...], type_name: Union[RuntimeType, str]) ->
     if len(inner_type_names) != len(val):
         raise TypeError("type_name members must have same length as tuple")
 
+    if inner_type_names == ['LogicalPlan', 'Expr']:
```

Comment:
In the rest of this function, the order doesn't seem to matter. If we saw `[ 'Expr', 'LogicalPlan']` would that just be a bug? Maybe compare as a set?

Reply:
I specifically only worked out FFI for the `(LogicalPlan, Expr)` tuple, in that order, because that is the carrier type for `ExprDomain`. I don't really see a need to add FFI for other variations at this point.